playground/mocks/data/idp/idx/enroll-profile-update-all-optional-params.json

```diff
@@ -37,7 +37,8 @@
                           "desc":"Use a second email to recover your account in case you become locked out. This email must be verified after set up.",
                           "required":false,
                           "minLength":1,
-                          "maxLength":100
+                          "maxLength":100,
+                          "descKey": "secondary.email.desc"
```

Comment:
Does `descKey` get used? Never seen it before

Reply:
Yeah that would be the part of other jira ticket i have.. It has been added in the endpoint, but not yet used in widget.

Reply:
I think you're adding it to reduce mock drift, just curious if the UI uses it